src/sentry/snuba/subscriptions.py

```diff
@@ -168,7 +169,9 @@ def _create_in_snuba(project, dataset, query, aggregation, time_window, resoluti
                 # We only care about conditions here. Filter keys only matter for
                 # filtering to project and groups. Projects are handled with an
                 # explicit param, and groups can't be queried here.
-                "conditions": get_filter(query).conditions,
+                "conditions": resolve_discover_aliases(
+                    {"conditions": get_filter(query).conditions}
+                )[0]["conditions"],
```

Comment:
Do you have any need to support aggregate conditions like `count():>10`? Those currently come out of `get_filter()` as a having property.

Reply:
No, at least for the moment we only care about actual fields.

Reply:
Should you raise an error if the user's conditions include aggregate conditions?

Reply:
There are various things we need to guard against, like date queries, queries against aggregates, etc. I don't think I'll tackle that here, but this will be handy for checking aggregates.